cmd/tether/main_linux.go

```diff
@@ -123,14 +123,22 @@ func halt() {
 
 func startSignalHandler() {
 	sigs := make(chan os.Signal, 1)
-	signal.Notify(sigs, syscall.SIGHUP)
+	signal.Notify(sigs, syscall.SIGHUP, syscall.SIGUSR1, syscall.SIGUSR2, syscall.SIGPWR, syscall.SIGTERM, syscall.SIGINT)
 
 	go func() {
 		for s := range sigs {
 			switch s {
 			case syscall.SIGHUP:
 				log.Infof("Reloading tether configuration")
```

Comment:
nit: could we add `via signal %s` to this log message for consistency? (Someone looking at the code can see that there's only one signal that generates this log line, but I think it's better not to assume everyone looking at the log messages is also reading the code.)